corelib-search/src/main/java/eu/europeana/corelib/search/impl/SearchServiceImpl.java

```diff
@@ -164,9 +167,14 @@ public FullBean processFullBean(FullBean fullBean, String europeanaObjectId, boo
     }
 
     public FullBean processFullBean(FullBean fullBean){
+        //check if manifestAddUrl property is present
         // add meta info for all webresources
-        WebMetaInfo.injectWebMetaInfoBatch(fullBean, mongoServer, manifestAddUrl);
-
+        if(manifestAddUrl==null || manifestAddUrl.equals(Boolean.FALSE)){
+            WebMetaInfo.injectWebMetaInfoBatch(fullBean, mongoServer, false, api2BaseUrl);
+        }
+        else {
+            WebMetaInfo.injectWebMetaInfoBatch(fullBean, mongoServer, true, api2BaseUrl);
+        }
```

Comment:
Here we check if manifestAddUrl == null, but in line 235 we do not and simply pass the value of manifestAddUrl (so this probably could cause a nullpointer exception?)
I think we should be consistent; either we check it also in line 235, or we remove the null-check here and move it to the WebMetaInfo.injectWebMetaInfoBatch method. My preference would be for the latter since then we only have to check the value in 1 location.